README.md

```diff
@@ -81,15 +81,35 @@ If you need more detailed instructions, see the [Novice setup guide][novice-setu
 [wiki]: https://github.com/cantino/huginn/wiki
 [novice-setup-guide]: https://github.com/cantino/huginn/wiki/Novice-setup-guide
 
+### Using Vagrant
```

Comment:
Like said above I would prefer it for Development. For checking out Huginn the docker image should be quicker. However we do not have an easy solution to get started developing on Huginn (especially on Windows) for which Vagrant would be perfect.